spec/unit/jobs/v3/services/update_broker_job_spec.rb

```diff
@@ -201,6 +201,30 @@ module V3
             end
           end
 
+          describe 'when there is a model level validation error i.e. blank credentials' do
+            it 'fails to update and raises InvalidServiceBroker' do
+              update_broker_request = ServiceBrokerUpdateRequest.create(
+                authentication: '{"credentials":{"username":"","password":""}}',
+                service_broker_id: broker.id
+              )
+              job = UpdateBrokerJob.new(update_broker_request.guid, broker.guid, previous_state, user_audit_info: user_audit_info)
```

Comment:
You should use `let(:update_broker_request)` in this context (describe); then the line `job = UpdateBrokerJob.new...` can be omitted.

I see that it is done wrong throughout the whole file, but we should not stick to this bad pattern.

Reply:
True, that looks better